### PR TITLE
[MOB-1250] parallelize tests

### PR DESCRIPTION
**Pull request by roninopf**

### CODE REVIEW

File: swift-sdk.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -867,6 +876,7 @@
 				AC98294A20D9D65E00796DAA /* Foundation.framework */,
 			);
 			name = Frameworks;
+			path = ..;
```
Comment (tapashmajumder) [inline swift-sdk.xcodeproj/project.pbxproj]: Why is this needed?

File: swift-sdk.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -849,7 +858,7 @@
 			path = "host-app";
 			sourceTree = "<group>";
 		};
-		ACFCA72920EB02DB00BFB277 /* Tests */ = {
```
Comment (tapashmajumder) [inline swift-sdk.xcodeproj/project.pbxproj]: We will need to change `Tests` to `tests` codecov.yml where we ignore this directory.

File: swift-sdk.xcodeproj/project.pbxproj (hunk 1 above)
Comment (roninopf) [reply swift-sdk.xcodeproj/project.pbxproj]: This was to put `Frameworks` and `Products` under the `misc` folder, just for slightly better organization.

**Comment by vbabenkoru** [comment]
Looks like the test aren't completing faster than without parallelization?
